File: demo/src/main/java/com/example/android/entity/Product.java
package com.example.android.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

import java.util.Date;

@Data
@TableName("products")
public class Product {
    @TableId(type = IdType.AUTO)
    private Integer id;
    private Long sellerId;
    private String title;
    private String description;
    private Integer categoryId;
    private Double price;
    private String images;
    private Integer status;
    private Integer viewCount;
    private String overallCondition;
    private Date createdAt;
    private Date updatedAt;
}

File: app/src/main/java/com/example/xiaoyi/model/Product.kt
package com.example.xiaoyi.model

data class Product(
    val id: Long,
    val name: String,//商品名
    val price: Double,//商品价格
    val description: String,//商品描述
    val location: String,//商品位置
    val imageUrl: String,//商品图片
    val userId: Long,//卖家ID
    val createdAt: String,//商品发布时间
    val sellerName: String = "未知卖家",//卖家名称
    val categoryId: Long = 0L//分类ID
)